cpp/src/barretenberg/honk/sumcheck/relations/relation_correctness.test.cpp

```diff
@@ -60,6 +61,145 @@ template <typename Flavor> void check_relation(auto relation, auto circuit_size,
     }
 }
 
+template <typename Flavor> void create_some_add_gates(auto& circuit_builder)
```

Comment:
For some reason the diff for this test file is hard to read. All I've done is (1) modularize the creation of various types of gates (arithmetic, lookup, etc.) into individual functions for reuse, and (2) add a relation correctness test for the GoblinUltraComposer which includes ECC op gates and checks the new op wire consistency relation in addition to the nominal Ultra Honk relations.